README.md

```diff
@@ -42,14 +42,9 @@ Logging will delegate any log requests to that provider.
       In an EE environment they are often already provided.
 
         <dependency>
-           <groupId>javax.el</groupId>
-           <artifactId>javax.el-api</artifactId>
-           <version>2.2.4</version>
-        </dependency>
-        <dependency>
```

Comment:
Nope. JBoss separated them in WildFly. They published the extracted API on Central but not the extracted impl. So I chose to use the official Glassfish jar which seems to be the norm out there.